src/Node/ClassPropertiesNode.php

```diff
@@ -355,6 +359,12 @@ private function getMethodsCalledFromConstructor(
 				continue;
 			}
 
+			if ($inMethod->getName() !== '__construct') {
```

Comment:
at this point we take the properties which have been assigned in `__construct` over to the `additionalConstructor` methods.